grails-app/services/com/redhat/theses/listeners/RoleRequestListenerService.groovy

```diff
@@ -0,0 +1,75 @@
+package com.redhat.theses.listeners
+
+import com.redhat.theses.RoleRequest
+import com.redhat.theses.Feed
+import com.redhat.theses.events.RoleRequestEvent
+import com.redhat.theses.util.Util
+import com.redhat.theses.auth.User
+import com.redhat.theses.auth.Role
+import grails.events.Listener
+import org.springframework.context.i18n.LocaleContextHolder as LCH
+
+class RoleRequestListenerService {
+
+    def messageSource
+    /**
+     * Dependency injection of com.redhat.theses.SubscriptionService
+     */
+    def subscriptionService
+
+    /**
+     * Dependency injection of org.codehaus.groovy.grails.web.mapping.LinkGenerator
+     */
+    def grailsLinkGenerator
+
+    /**
+     * Creates new feed and notifies owner of applied topic about creation of new application
+     */
+    @Listener(topic = "RoleRequestCreated")
+    def RoleRequestCreated(RoleRequestEvent e) {
```

Comment:
I only changed method names to start with lowercase. I didn't change anything else, because I wanted it to be consistent across all Listener classes.